src/txkube/test/test_swagger.py

```diff
@@ -567,14 +572,15 @@ def test_array_property(self):
         # Arbitrarily select a simple definition that includes an array from
         # the spec.  It also demonstrates that "required" itself is optional.
         name = u"v1.Capabilities"
+        capability = b"hello"
         Capabilities = spec.pclass_for_definition(name)
 
         self.expectThat(
-            lambda: Capabilities(add=b"hello"),
+            lambda: Capabilities(add=capability),
             raises_exception(
                 CheckedValueTypeError,
                 expected_types=(unicode,),
-                actual_type=bytes,
+                actual_type=type(capability[0]),
```

Comment:
Might be neater to stop using a byte string as the bogus value now... A list of int or some such would avoid the need to care about the different indexing behavior on Python 2/3.